frog/settings.json

```diff
@@ -1,10 +1,3 @@
 {
-  "Minio": {
-    "urls": [
-      "https://icchilisrv1.epfl.ch:9000",
-      "https://icchilisrv1.epfl.ch:9001"
-    ],
-    "accessKey": "AKIAIOSFODNN7EXAMPLE",
-    "secretKey": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
-  }
+  "token": "aab"
```

Comment:
Why do you fill a token here rather than by default using the random generation?